src/components/Editpost/Editpost.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+import {Button,TextArea,Modal,Input, Segment, Container,Menu,Icon,Sidebar, Header} from 'semantic-ui-react';
+import Validation from '../Validation/Validation';
+import Axios from '../../axios-orders';
+import {Link} from 'react-router-dom';
+import {connect} from 'react-redux';
+class editPost extends Component{
+    state={
+        loadedData:'null'
+    }
+    componentDidMount(){
+        Axios.get(`posts/${this.props.match.params.id}.json`)
+        .then(response=>{
+            console.log(response.data);
+            this.setState({loadedData:{...response.data}});
+            console.log(this.state.loadedData);
+        });
+    }
+    inputHandler=(event)=>{
+        let updatedPost={...this.state.loadedData};
+        updatedPost[event.target.name]=event.target.value;
+        this.setState({loadedData:updatedPost});
+        console.log(this.state.loadedData);
+    }
+    savedUpdatedDataToServerHandler=()=>{
+        let valid=false;
+        valid=Validation(this.state.loadedData);
+        console.log(valid);
+        if(valid){
+        Axios.patch(`posts/${this.props.match.params.id}.json`,this.state.loadedData)
+    .then( response => {
+         console.log( response.data );
+        alert("Successfully updated...");
```

Comment:
After successfully update of `post` it should be redirected to the `post` list page.

Reply:
SIr,done with this part also